buildscripts/templates/SemanticAttributes.java.j2

```diff
@@ -74,42 +68,23 @@ public final class {{class}} {
         <ul> {{attribute.note | replace("> ", "") | render_markdown(code="{{@code {0}}}", paragraph="<li>{0}</li>", list="{0}")}} </ul>
 
     {%- endif %}
-    {%- if (attribute.stability | string())  == "StabilityLevel.DEPRECATED" %}
+    {%- if attribute | is_deprecated %}
   *
   * @deprecated {{attribute.brief | to_doc_brief}}.
```

Comment:
I *think* this should fix deprecated annotations and work for both 1.23.1 and 1.24.0

```suggestion
  * @deprecated {{ attribute.deprecated if attribute.deprecated else attribute.brief | to_doc_brief}}.
```

Reply:
ah, wait, the problem is in `attribute | is_deprecated`, so  to make it work with 1.23.x and 1.24.0 would be to change it to `if attribute.deprecated` (`deprecated` is a string)